### ISSUE TITLE
Authorization feature

### ISSUE THREAD

**Issue by Pinevskiy** [oxlip-lang/oal#14]
Generating specification for authorization is not possible or is not in documentation. 
Would be usefull to have ability to create "securitySchemas" and specify in methods if authorization is needed.

**Comment by ebastien** [comment]
Agreed. The focus has been on covering API endpoints and schemas. I relied on the so-called ["base" OpenAPI description](https://www.oxlip-lang.org/doc/usage.html) to cover things that the language doesn't support yet. The base document is merged at the JSON/YAML level with the description generated by Oxlip. It is far from perfect and a bit hacky. I fully agree with that.
Hopefully, that is something I can address in the future, but I am also happy to get contributions!
